code/modules/mob/living/carbon/life.dm

```diff
@@ -154,9 +154,9 @@
 			adjustToxLoss(4)
 
 	switch(drunkenness) //painkilling effects
-		if(51 to 71)
+		if(6 to 41)
```

Comment:
99% sure gamers are going to just powergame this in their mixes for extra pain resist.

Reply:
its only light and medium so if they really want to they could try and manage alcohol levels or they could just use inap or synap instead for a much easier 25 pain resist

Reply:
> 99% sure gamers are going to just powergame this in their mixes for extra pain resist.

pain resist? more like, lame insist! Running Vali, the most loadout friendly primary, the only time i seriously looked at including alcohol for any of their effects is when the PR for putting chems effects behind a delay after getting hit was open. Even when i was running BKTTOxy (-150) i didn't notice much of a difference from just Tram (-50) or even Para(-35), painkill is such a small thing that i find, when possible, it's better to just run small dose BKMDTri (0 painkill) instead of BKTT or any other variation.